site/source/docs/tools_reference/emsdk.rst

```diff
@@ -90,27 +90,17 @@ Below are typical **.emscripten** files created by *emsdk*. Note the variable na
   # .emscripten file from Windows SDK
 
   import os
-  SPIDERMONKEY_ENGINE = ''
-  NODE_JS = 'node'
```

Comment:
why is node removed?

Reply:
Because its also listed below .. for some reason it appeared twice here.